CMakeLists.txt

```diff
@@ -85,6 +85,7 @@ xoption(BUILD_STATIC_QJS_EXE "Build a static qjs executable" OFF)
 xoption(CONFIG_ASAN "Enable AddressSanitizer (ASan)" OFF)
 xoption(CONFIG_MSAN "Enable MemorySanitizer (MSan)" OFF)
 xoption(CONFIG_UBSAN "Enable UndefinedBehaviorSanitizer (UBSan)" OFF)
+xoption(HAVE_TM_GMTOFF "Enable when `tm_gmtoff` exists on `struct tm`" ON)
```

Comment:
I thought we were going to drop this?

Reply:
It's an explicit option to disable, defaults to `ON`.

Reply:
Yes but if you just copy the C files over to a project with an existing build system it won't work.

Ideally updating from QuickJS to our NG is a simple as overwriting the files, without the need to add extra build time flags.